docs/ops/activation/PReLU_1.md

```diff
@@ -32,13 +32,13 @@ PReLU(x) = \max(0, x) + \alpha\cdot\min(0, x)
 
 **Inputs**
 
-* **1**: `data`. A tensor of type `T` and arbitrary shape. **Required**.
-* **2**: `slope`. 1D tensor of type `T`. Tensor with negative slope values, one per channel dimension of `data` input tensor. **Required**.
-* **Note**: Channels dimension corresponds to second dimension of `data` input tensor. If `data` rank is less than 2, the number of channels is 1.
+* **1**: `data`. A tensor of type *T* and arbitrary shape. **Required**.
+* **2**: `slope`. A tensor of type *T* and rank greater or equal to 1. Tensor with negative slope values. **Required**.
+* **Note**: Channels dimension corresponds to the second dimension of `data` input tensor. If `slope` input rank is 1 and its dimension is equal to the second dimension of `data` input, then per channel broadcast is applied. Otherwise `slope` input is broadcasted with numpy rules.
```

Comment:
Please, add a link to the broadcasting rules: `[Broadcast Rules For Elementwise Operations](../broadcast_rules.md)`

Reply:
Thank you, done.